test/caesar_test.rb

```diff
@@ -1,2 +1,32 @@
+require 'minitest/pride'
+require 'minitest/autorun'
+require './lib/caesar.rb'
+
+class CaesarTest < MiniTest::Test
+
+  def test_it_exits
+    caesar = Caesar.new("the quick brown fox jumps over the lazy dog")
+    assert_instance_of Caesar, caesar
+  end
+
+  def test_translate_english_to_ciphertext
+    caesar = Caesar.new("the quick brown fox jumps over the lazy dog")
+    assert_equal "the quick brown fox jumps over the lazy dog", caesar.eng_to_cipher
```

Comment:
You are correct that caesar should have a method called `eng_to_cipher`, but if you look at the spec if should take two parameters, the text and the key. Also, you should be asserting the return value is the ciphertext "qeb nrfzh yoltk clu grjmp lsbo qeb ixwv ald", not the original text.